GenerateJSCode.php

```diff
@@ -230,12 +230,14 @@ public function generateInitialJSCode($datasource, $options, $dbspecification, $
 
         $this->generateAssignJS(
             "INTERMediatorOnPage.isNativeAuth",
-            (isset($options['authentication']) && isset($options['authentication']['user'])
-                && ($options['authentication']['user'][0] === 'database_native')) ? "true" : "false");
+            ((isset($options['authentication'])
+                && isset($options['authentication']['user'])
+                && ($options['authentication']['user'][0] === 'database_native'))
+                || ((new LDAPAuth())->isActive)) ? "true" : "false");
```

Comment:
Oops! This issue is described below. new xx -> isn't work under 5.3.
http://stackoverflow.com/questions/13388541/php-parse-error-syntax-error-unexpected-t-object-operator